README.md

```diff
@@ -15,18 +14,22 @@ This document is for sites running on VIP Go.
 
 ## Overview
 
-VIP Go allows you to differentiate visitors from different countries, tailoring the content you serve to your visitors on a country by country basis, while still keeping the benefits of our fast caching infrastructure.
+The features of the VIP Go Geo Uniques plugin are designed to work on a [VIP Platform environment](https://docs.wpvip.com/technical-references/vip-platform/environments/) and will not work as expected on a local development environment.
 
-Adding geo targeting to your VIP Go code takes two parts:
+This plugin can be used on a non-VIP platform host infrastructure, but only if the host infrastructure has the `$_SERVER['GEOIP_COUNTRY_CODE']` superglobal.
+
+Sites running on the WordPress VIP Platform with the  VIP Go Geo Uniques plugin can differentiate visitors from different countries. This plugin makes it possible to tailor the content you serve to your visitors on a country by country basis, while still keeping the benefits of VIP's fast caching infrastructure.
+
+Adding geo targeting to your VIP code takes two parts:
 
 1. Loading and configuring the VIP Go Geo Uniques plugin
 2. Serving geo targeted content
 
 ## Loading the VIP Go Geo Uniques plugin
 
-As the VIP Go Geo Uniques plugin requires configuration in code, we recommend loading this plugin using the `wpcom_vip_load_plugin` function.
+Because the VIP Go Geo Uniques plugin requires configuration in code, we recommend [loading this plugin with the function](https://docs.wpvip.com/how-tos/activate-plugins-through-code/#plugins-in-the-plugins-directory) `wpcom_vip_load_plugin` function.
```

Comment:
```suggestion
Because the VIP Go Geo Uniques plugin requires configuration in code, we recommend [activating this plugin](https://docs.wpvip.com/how-tos/activate-plugins-through-code/#plugins-in-the-plugins-directory)  with the `wpcom_vip_load_plugin()` function.
```